theme/jade/index-partials/tickets.jade

```diff
@@ -5,9 +5,9 @@ section.section.section--tickets.tickets
     .section__copy
       dl.tickets__prices
         dt Early Bird:
-        dd <span class='bold'>€59</span> (Sold out!)
+        dd  <span class='bold'>€59</span> (Sold out!)
```

Comment:
Because else the `:` and the price stick together. First space is the normal space to separate a element from the content, the second space is part of the content. Currently it looks like this:
<img width="419" alt="bildschirmfoto 2016-02-13 um 18 21 05" src="https://cloud.githubusercontent.com/assets/840022/13028962/973a1c02-d27e-11e5-9172-2aa9c8c352ad.png">